Automattic/wp-calypso
# Hosting metrics: Update chart requests resolution using _meta

Related to #

## Proposed Changes

* Replace `Requests per minute` with `Requests per second` 
* Use `_meta.resolution` to calculate requests per second

## Testing Instructions



* Navigate to `/site-monitoring/${siteSlug}`
* Observe the line charts display the label `Requests per second`, and the values are higher because we are using the resolution.

## Screenshot
<img width="935" alt="Screenshot 2023-08-17 at 14 28 36" src="https://github.com/Automattic/wp-calypso/assets/779993/3b09d168-74e3-4fbd-ac2b-4b249bbd1259">

<img width="939" alt="Screenshot 2023-08-17 at 14 28 41" src="https://github.com/Automattic/wp-calypso/assets/779993/3b3a6e71-4e3d-4fba-81c6-0f87326f2332">

<img width="932" alt="Screenshot 2023-08-17 at 14 28 47" src="https://github.com/Automattic/wp-calypso/assets/779993/d209259c-0476-4481-b013-35a81add0c39">



## Pre-merge Checklist
